src/css/Navbar.css

```
@@ -29,152 +48,127 @@
   }
 }
 
-#navbar-links {
-  display: flex;
-  flex-direction: row;
-  flex-grow: 1;
-  justify-content: flex-end;
-  align-items: center;
-  padding-right: 10px;
-  margin: 0;
-}
-
-#navbar-links a,
-#navbar-links a:visited {
-  text-decoration: none;
-  color: var(--offwhite);
-}
-
-#navbar-links li {
-  padding: 0 15px;
-  height: 45px;
-  line-height: 45px;
-  list-style-type: none;
-  position: relative;
-  display: inline-block;
-  margin: 5px 10px;
-  outline: none;
-  color: var(--offwhite);
-  text-decoration: none;
-  letter-spacing: 1px;
-  font-weight: 400;
-  text-shadow: 0 0 1px rgba(255, 255, 255, 0.3);
-  font-size: 0.75em;
-}
-
-@media only screen and (max-width: 800px) {
-  #navbar-links li {
-    height: 20px;
-    font-size: .5em;
-    line-height: 25px;
-    padding: 0 7px;
+/* When the screen is less than 600 pixels wide, hide all links, except for the first one ("Home"). Show the link that contains should open and close the topnav (.icon) */
+@media screen and (max-width: 600px) {
+  .nav-links li:not(:first-child) {display: none;}
+  .nav-links li.icon {
+    float: right;
+    display: block;
   }
 }
 
-#navbar-links li::before {
-  top: 0;
-  left: 0;
-  -webkit-transform: rotate(90deg);
-  -moz-transform: rotate(90deg);
-  transform: rotate(90deg);
-  -webkit-transform-origin: 0 0;
-  -moz-transform-origin: 0 0;
-  transform-origin: 0 0;
+/* The "responsive" class is added to the topnav with JavaScript when the user clicks on the icon. This class makes the topnav look good on small screens (display the links vertically instead of horizontally) */
+@media only screen and (max-width: 600px) {
+  .nav-links.responsive {position: relative;}
+  .nav-links.responsive a.icon {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  .nav-links.responsive a {
+    float: none;
+    display: block;
+    text-align: left;
+  }
 }
 
-#navbar-links li::after {
-  right: 0;
-  bottom: 0;
-  -webkit-transform: rotate(90deg);
-  -moz-transform: rotate(90deg);
-  transform: rotate(90deg);
-  -webkit-transform-origin: 100% 0;
-  -moz-transform-origin: 100% 0;
-  transform-origin: 100% 0;
+.desktop-nav {
+  display: none;
```

You could also do this with `visibility` instead, so you don't have to worry about making it `display: block` vs `display: flex` when it's unhidden. Not a necessary change, but might be easier.
```suggestion
  visibility: hidden;
```

---

I choose display over visibility, because visibility hidden will still affect the layout. In case someone will change the mobile navbar in the future, I don't want them to have to deal with the desktop nav ruining anything. But good suggestion :)